scripts/legacy2luatest.pl

```diff
@@ -133,8 +133,15 @@ sub read_in_file {
         # If line contains single quotes or backslashes, use double
         # square brackets to wrap string.
         if (/'/ || /\\/) {
-          $startstr = '[[';
-          $endstr = ']]';
+            # If line contains ending square bracket, use square brackets
```

Comment:
It doesn't matter if an opening bracket exists or not, the closing bracket can still cause a premature end of string. But I will reword the comment, thanks.